clarity_ext/service/file_service.py

```diff
@@ -267,6 +270,14 @@ def __init__(self, file_service, file_repo, artifact_service, downloaded_path, o
         self.should_cache = should_cache
         self.logger = logger
 
+    def list_filenames(self, file_handle):
+        shared_files = self.artifact_service.shared_files()
+        by_handle = [
+            shared_file for shared_file in shared_files
+            if shared_file.name == file_handle
+        ]
+        return [a.files[0].original_location for a in by_handle]
```

Comment:
Do you know if `original_location` returns the path and the file name? I don't remember this one (the name indicates only a path to me.)

Reply:
I checked now (in integration tests). It seems to be only the file name.

Reply:
Ok!